landoapi/landing_worker.py

```diff
@@ -168,7 +166,7 @@ def refresh_enabled_repos(self):
         self.enabled_repos = [
             r
             for r in self.applicable_repos
-            if treestatus_subsystem.client.is_open(repo_clone_subsystem.repos[r].tree)
+            if treestatus.is_open(repo_clone_subsystem.repos[r].tree)
```

Comment:
Would be nice to have something like:
```python
tree = Tree.query.get(name=repo_clone_subsystem.repos[r].tree)
if tree.is_open():
    # ... do stuff
```

Reply:
A class method would be nice, so the query details are close to the model definition: `Tree.is_open(repo_clone_subsystem.repos[r].tree)`

Reply:
Anything stopping us from actually including the database trees as part of `repo_clone_subsystem.repos`? I.e., mapping `repo_clone_subsystem.repos[r].lando_tree` to a `Tree` object, or something similar.

Reply:
Tying the `Tree` model to a `Repo` is something I would like to do, since they are mostly referring to the same concept. I'll take a look at trying to do this, but unless we can think of a technical reason that these models couldn't be merged later, I don't think it's a hard blocker for this project.